---
layout: page
title: Lighthouse project
---

## Development and validation of a multi-omics workflow to diagnose rare diseases

<div class="row">
    <div class="6u 12u$(small)">
        <h3>Challenges in diagnosing rare diseases</h3>
        Rare diseases impact approximately 400 million people globally, with 500,000 in Switzerland alone, often leading to premature death or chronic disability, particularly in children. The identification of genetic and biochemical markers through advanced genomic technologies is crucial to enhance diagnostic accuracy and treatment strategies. Our approach integrates multi-omics data—combining genomic, transcriptomic, proteomic, and metabolomic information—to uncover relationships between genotype and phenotype, significantly improving the diagnostic yield for severe rare diseases.
    </div>

    <div class="6u 12u$(small)">
        <h3>Advancing personalised medicine through multi-omics</h3>
        Multi-omics technologies allow for a comprehensive analysis that extends beyond traditional whole genome sequencing (WGS), offering deeper insights into the complex biology of rare diseases. By integrating these technologies, we can detect rare genomic variants and their functional impacts, facilitating rapid and precise medical interventions. This integrated approach is particularly effective in diagnosing children with extreme phenotypes, enabling timely and potentially lifesaving treatments.
    </div>
</div>

<hr>
## Gaps in current practice

Despite advances, current diagnostic processes such as whole exome sequencing (WES) and WGS often lead to delays and high rates of undiagnosed cases. Our Lighthouse project addresses these issues by:
1. Reducing diagnostic delays through the integration of streamlined multi-omics workflows.
2. Increasing diagnostic yield by employing advanced bioinformatics tools that enhance the analysis of complex data sets.
3. Extending multi-omics applications from research environments to clinical settings, thus broadening their diagnostic utility.

<hr>
## Project phases

<span style="color: green; font-size: 150%;">&#x2611;</span>
**Phase 1 Complete:**
The initial phase involved deploying a comprehensive multi-omics approach to diagnose and characterise individuals with rare metabolic disorders, achieving a significant increase in diagnostic rates.

<span style="color: green; font-size: 150%;">&#x2610;</span>
**Phase 2 Ongoing:**
Currently, we are expanding our multi-omics strategy to include larger cohorts, employing integrated genomic, transcriptomic, and metabolomic analyses to identify novel disease-causing variants.

<span style="color: green; font-size: 150%;">&#x2610;</span>
**Phase 3 Ongoing:**
The next phase will implement these approaches in a prospective study, enhancing our understanding of the clinical utility of multi-omics data and improving patient and family experiences with genetic diagnoses.

<hr>
## Research design

### Patient recruitment and phenotype selection

**Phase 2:** 
We access existing cohorts comprising 250 children with extreme phenotypes, already consented for the use of stored biospecimens. This includes 150 children with severe metabolic dysfunction, categorised into those without molecular data, those with Variants of Uncertain Significance (VUS), and undiagnosed patients. Additionally, 100 children suspected of primary immunodeficiency due to severe infections or immune dysregulation will be studied.

**Phase 3:** 
In a multi-site study, we will recruit a prospective cohort of 140 patients from intensive care settings. We will collect blood samples, skin biopsies, and mucosal swabs for comprehensive genomic, transcriptomic, proteomic, and metabolomic analysis. Patients will be selected based on specific criteria, including acute encephalopathy, severe metabolic dysfunction, and unusual severe infections or inflammation syndromes.

### Clinical data processing

We will use a core set of data parameters to capture clinical features and outcomes, integrating this with electronic health record (EHR) data to characterize patient phenotypes comprehensively. This will also include enriching our dataset with high-resolution omics data, aiming to enhance national data systems and establish a continuum of data that tracks patient outcomes over time.

### Multi-omics data generation 

**Phase 2 and Phase 3:** 
Patient-derived biospecimens will be processed to extract DNA, RNA, proteins, and metabolites. These samples will undergo:

- **Nucleic Acid-Based Analyses:** Whole genome sequencing (WGS) and RNA sequencing (RNA-seq) will be conducted to generate high-quality data that aids in clinical decision-making.
- **Proteomic Profiling:** Advanced proteomics technologies, such as DIA-MS, will be utilized to convert clinical specimens into detailed proteotype maps.
- **Metabolomics and Lipidomics Analyses:** These will be performed using high-throughput technologies to analyze primary metabolites and xenobiotics, providing a broad overview of the biochemical state of the samples.

### Primary data analyses

Advanced bioinformatics tools will be applied to perform automated DNA variant calling and annotation. We will prioritize rare variants based on their potential pathogenicity and relevance to the clinical presentation. RNA-seq data will undergo quantitative expression profiling and splicing analysis. We will integrate and visualize genetic, transcriptomic, proteomic, and metabolomic data to provide a comprehensive multi-dimensional view of patient data, aligning with the latest FAIR data principles for enhanced data utility and accessibility.

### Multi-omics results using multimodal machine learning analysis

**Phase 2:** 
We will employ machine learning techniques to analyze integrated multi-omics and clinical data, focusing on a cohort of children with fully annotated datasets. This approach will help us identify critical biomarkers and patterns that are predictive of rare diseases.

**Phase 3:** 
Our validated machine learning models will be further optimized for diagnostic use in clinical settings, providing probabilistic assessments of disease phenotypes to support clinical decision-making. This phase will also explore the enrichment of our models with additional genomic data from family trios or functional datasets to enhance diagnostic accuracy.

### Implementation of findings

Findings from the project will be directly translated into clinical practice, with an emphasis on improving the diagnostic yield and informing treatment strategies. We will also engage with medical professionals and patient advocacy groups to ensure that our research aligns with patient needs and ethical standards.

This comprehensive approach ensures a robust framework for diagnosing and understanding rare diseases through advanced multi-omics analyses, supported by cutting-edge bioinformatics and statistical methodologies.


<hr>
## Scientific output and expected contributions

<div class="row">
    <div class="6u 12u$(small)">
        <h3>Development of a novel multi-omics workflow</h3>
        <p>This project enhances current genetic testing methods by integrating multiple data streams, offering comprehensive insights that accelerate rare disease diagnostics and therapeutic developments.</p>
    </div>

    <div class="6u 12u$(small)">
        <h3>Identification of novel diagnoses and pathomechanisms</h3>
        <p>Through the analysis of both existing and new patient cohorts, our project identifies novel disease mechanisms, demonstrating the power of multi-omics to uncover underlying causes of rare conditions.</p>
    </div>
</div>

