<!DOCTYPE HTML>
<!--
	Editorial by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>

<head>
  <title>SwissPedHealth</title>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
  <!--[if lte IE 8]><script src="/assets/js/ie/html5shiv.js"></script><![endif]-->
  <link rel="stylesheet" href="/assets/css/main.css" />
	<script type="text/javascript" async
		src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/latest.js?config=TeX-MML-AM_CHTML">
	</script>
</head>


<body>

    <!-- Wrapper -->
    <div id="wrapper">

        <!-- Main -->
        <div id="main">
            <div class="inner">

                
<header id="header">
	<div id="header-icons">
		<ul class="icons">
			<li><a href="/" class="icon"><span class="label">Home</span></a></li>
			<li><a href="about.html" class="icon"><span class="label">About</span></a></li>
			<li><a href="/index.html#news" class="icon"><span class="label">News</span></a></li>
			<li><a href="/index.html#projects" class="icon"><span class="label">Projects</span></a></li>
			<!-- Additional icons here -->
		</ul>
	</div>

	<!-- <div id="header-logo"> -->
	<!-- 	<div class="title-subtitle-block"> -->
	<!-- 		<a href="/" class="logo"><strong>Switzerland</strong><sp> Omics</a> -->
	<!-- 	</div> -->
	<!-- </div> -->

<div id="header-logo">
  <div class="title-subtitle-block">
    <a href="/" class="logo">
      <!-- <span class="title"><strong>Switzerland</strong></span> -->
      <span class="title">Switzerland</span>
      <span class="subtitle">&nbsp;Omics</span>
    </a>
  </div>
</div>


  <!-- <div id="header-logo"> -->
  <!--   <a href="/" class="logo"> -->
  <!--     <!-1- <img src="assets/images/logos/swiftlogo.webp" alt="Logo" class="logo-image"> -1-> -->
  <!--     <span class="title-subtitle-block"> -->
  <!--       <span class="title">Switzerland</span> -->
  <!--       <span class="subtitle">Omics</span> -->
  <!--     </span> -->
  <!--   </a> -->
  <!-- </div> -->
 
 

</header>




                <!-- Content -->
                <section>
                	<header class="main">
                		<h1>Sequence Kernel Association Test</h1>
                	</header>

                  

                <h2 id="data-notation">Data Notation</h2>

<ul>
  <li>Assume \(n\) subjects are sequenced in a region with ( p ) variant sites observed.</li>
  <li>Covariates might include age, gender, and top principal components of genetic variation for controlling population stratification.</li>
  <li>For the ( i )-th subject,
    <ul>
      <li>( y_i ) denotes the phenotype variable,</li>
      <li>( X_i = (X_{i1}, X_{i2}, …, X_{im}) ) the covariates,</li>
      <li>( G_i = (G_{i1}, G_{i2}, …, G_{ip}) ) the genotypes for the ( p ) variants within the region.</li>
    </ul>
  </li>
  <li>Typically, we assume an additive genetic model and let ( G_{ij} = 0, 1, ) or ( 2 ) represent the number of copies of the minor allele.</li>
  <li>Dominant and recessive models can also be considered.</li>
</ul>

<p>For example, let’s consider that we have sequenced 200 subjects and only have 10 variant sites in a particular genomic region. We have three covariates: age, gender, and the first principal component of genetic variation (PC1).</p>

<div class="language-markdown highlighter-rouge"><div class="highlight"><pre class="highlight"><code>| Subject | Phenotype (<span class="se">\(</span> y_i <span class="se">\)</span>) | Covariates (<span class="se">\(</span> X_i <span class="se">\)</span>)      | Genotypes (<span class="se">\(</span> G_i <span class="se">\)</span>)                |
|---------|-----------------------|-----------------------------|--------------------------------------|
| 1       | <span class="se">\(</span> y_1 <span class="se">\)</span>             | <span class="se">\(</span> X_{11}, X_{12}, ..., X_{1m} <span class="se">\)</span> | <span class="se">\(</span> G_{11}, G_{12}, ..., G_{1p} <span class="se">\)</span> |
| 2       | <span class="se">\(</span> y_2 <span class="se">\)</span>             | <span class="se">\(</span> X_{21}, X_{22}, ..., X_{2m} <span class="se">\)</span> | <span class="se">\(</span> G_{21}, G_{22}, ..., G_{2p} <span class="se">\)</span> |
| ...     | ...                   | ...                         | ...                                  |
| <span class="se">\(</span> i <span class="se">\)</span> | <span class="se">\(</span> y_i <span class="se">\)</span>             | <span class="se">\(</span> X_{i1}, X_{i2}, ..., X_{im} <span class="se">\)</span> | <span class="se">\(</span> G_{i1}, G_{i2}, ..., G_{ip} <span class="se">\)</span> |
| ...     | ...                   | ...                         | ...                                  |
| <span class="se">\(</span> n <span class="se">\)</span> | <span class="se">\(</span> y_n <span class="se">\)</span>             | <span class="se">\(</span> X_{n1}, X_{n2}, ..., X_{nm} <span class="se">\)</span> | <span class="se">\(</span> G_{n1}, G_{n2}, ..., G_{np} <span class="se">\)</span> |
</code></pre></div></div>

<p><em>Table: Tabular representation of the data for ( n ) subjects, each with their phenotype, covariates, and genotypes for ( p ) variants within the region.</em></p>

<h2 id="linear-skat-model-and-test">Linear SKAT Model and Test</h2>

<p>To analyze this data using a statistical model, we consider the phenotype as a continuous trait, employing a linear model described by:</p>

<p>[
y_i = \alpha_0 + \alpha’X_i + \beta’G_i + \epsilon_i
]</p>

<ul>
  <li>( \alpha_0 ) is an intercept term,</li>
  <li>( \alpha ) is the vector of regression coefficients for the covariates,</li>
  <li>( \beta ) is the vector of regression coefficients for the genetic variants in the region.</li>
</ul>

<p>For dichotomous traits like blood pressure (high or normal), a logistic model is used:</p>

<p>[
\text{logit} P(y_i = 1) = \alpha_0 + \alpha’X_i + \beta’G_i
]</p>

<p>Here, ( P(y_i = 1) ) represents the probability of the ( i )-th subject having high blood pressure.</p>

<h3 id="skat-analysis">SKAT Analysis</h3>

<p>SKAT improves test power by assuming each ( \beta_j ) follows an arbitrary distribution with mean zero and variance ( w_j\tau ), where ( \tau ) is a variance component and ( w_j ) is a pre-specified weight for variant ( j ).</p>

<p>The SKAT model tests whether the gene variants influence the phenotype, adjusting for covariates, corresponding to the null hypothesis ( H_0: \beta = 0 ).</p>

<h3 id="example-calculation">Example Calculation</h3>

<p>Given a matrix ( G ) of genotypes and a diagonal matrix ( W ) of weights, we compute the kernel matrix ( K ) as follows:</p>

<p>[
K = GWG’
]</p>

<p>where ( K(G_i, G_j) = \sum_{k=1}^{p} w_k G_{ik} G_{jk} ) measures the genetic similarity between subjects based on their genotypes adjusted by variant weights.</p>

<p>This analytical framework allows for a robust examination of the influences of multiple genetic variants on diseases such as high blood pressure, providing insights that are crucial for advancing personalized  medicine.</p>



              </div>
          </div>

          <!-- Sidebar -->
  <div id="sidebar">
    <div class="inner">

      <!-- <!-1- Search -1-> -->
      <!--   <section id="search" class="alt"> -->
      <!--     <form method="post" action="#"> -->
      <!--       <input type="text" name="query" id="query" placeholder="Search" /> -->
      <!--     </form> -->
      <!--   </section> -->

      <!-- Menu -->
        <nav id="menu">
          <header class="major">
            <h2>Menu</h2>
          </header>
          <ul>
            <li><a href="/">Homepage</a></li>
            <li><a href="about.html">About</a></li>
            <!-- <li><a href="patient_checklist.html">Patient checklist</a></li> -->
            <li>
              <span class="opener">Projects</span>
              <ul>
                <li><a href="">Lighthouse</a></li>
              </ul>
            </li>
            <li><a href="/404.html">Funding</a></li>
            <li><a href="/index.html#news">In the news</a></li>
          </ul>
        </nav>

      <!-- Section -->
        <section>
          <header class="major">
            <h2>Highlights for patients</h2>
          </header>
          <div class="mini-posts">
            <article>
              <a href="/information_genomic_analysis.html" class="image"><img src="assets/images/national-cancer-institute-W6yy0wYV-hk-unsplash.jpg" alt="" /></a>
              <p>Genomic analysis allows our scientists to identify health concerns and discover new treatments.</p>
            </article>
          </div>
          <!-- <ul class="actions"> -->
          <!--   <li><a href="#" class="button">More</a></li> -->
          <!-- </ul> -->
        </section>

      <!-- Footer -->
        <footer id="footer">
			<p class="copyright">&copy; Switzerland. All rights reserved. Images: <a href="https://unsplash.com/license">license</a> and <a href="https://creativecommons.org/licenses/by-nc/4.0/legalcode">AI generated image license</a>. Design: <a href="https://html5up.net">HTML5 UP</a> Production: <a href="https://lawlessgenomics.com">Dylan Lawless</a>.</p>
        </footer>

    </div>
  </div>


      </div>

      <!-- Scripts -->
  <script src="assets/js/jquery.min.js"></script>
  <script src="assets/js/skel.min.js"></script>
  <script src="assets/js/util.js"></script>
  <!--[if lte IE 8]><script src="assets/js/ie/respond.min.js"></script><![endif]-->
  <script src="assets/js/main.js"></script>


</body>

</html>
