<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8"/>
<style>body{background-color:white;}</style>
<link href="common_lib/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet" />
<script src="common_lib/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="common_lib/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="common_lib/datatables-binding-0.33/datatables.js"></script>
<script src="common_lib/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="common_lib/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="common_lib/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="common_lib/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<script src="common_lib/jszip-1.13.6/jszip.min.js"></script>
<script src="common_lib/pdfmake-1.13.6/pdfmake.js"></script>
<script src="common_lib/pdfmake-1.13.6/vfs_fonts.js"></script>
<link href="common_lib/dt-ext-buttons-1.13.6/css/buttons.dataTables.min.css" rel="stylesheet" />
<script src="common_lib/dt-ext-buttons-1.13.6/js/dataTables.buttons.min.js"></script>
<script src="common_lib/dt-ext-buttons-1.13.6/js/buttons.html5.min.js"></script>
<script src="common_lib/dt-ext-buttons-1.13.6/js/buttons.colVis.min.js"></script>
<script src="common_lib/dt-ext-buttons-1.13.6/js/buttons.print.min.js"></script>
<link href="common_lib/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet" />
<script src="common_lib/crosstalk-1.2.1/js/crosstalk.min.js"></script>
  <title>datatables</title>
</head>
<body>
<div id="htmlwidget_container">
  <div class="datatables html-widget html-fill-item" id="htmlwidget-21ad53bf65548e180d4d" style="width:100%;height:auto;"></div>
</div>
<script type="application/json" data-for="htmlwidget-21ad53bf65548e180d4d">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"data":[["1","2","3","4","5","6","7","8"],[228,228,228,228,228,228,228,228],["APOA5","APOC2","CREB3L3","GPD1","GPIHBP1","LMF1","LPL","CELSR2"],["Severe hypertriglyceridaemia","Severe hypertriglyceridaemia","Severe hypertriglyceridaemia","Severe hypertriglyceridaemia","Severe hypertriglyceridaemia","Severe hypertriglyceridaemia","Severe hypertriglyceridaemia","Severe hypertriglyceridaemia"],["gene","gene","gene","gene","gene","gene","gene","gene"],["3","3","3","3","3","3","3","1"],["Complete","Complete",null,"Complete","Complete","Complete","Complete","Complete"],["","",null,"","","","",""],[["27604308","12417525","27678447","16200213","23307945"],[],["29954705","21666694","26427795","22135386"],["22226083","24549054"],[],["17994020","19820022","30885219","30420299","29910226","22239554"],[],[]],[["Expert Review Green","UKGTN","Radboud University Medical Center, Nijmegen","Eligibility statement prior genetic testing"],["UKGTN","Expert Review Green","Eligibility statement prior genetic testing"],["Expert Review Green","Literature"],["Expert Review Green","Radboud University Medical Center, Nijmegen"],["Expert Review Green","UKGTN","Eligibility statement prior genetic testing"],["Expert Review Green","UKGTN","Eligibility statement prior genetic testing"],["Expert Review Green","UKGTN","Illumina TruGenome Clinical Sequencing Services","Radboud University Medical Center, Nijmegen","Eligibility statement prior genetic testing"],["Other"]],[["Hyperchylomicronemia, late-onset OMIM:144650","hyperlipoproteinemia type V MONDO:0007762","{Hypertriglyceridemia, susceptibility to} OMIM:145750","hypertriglyceridemia, familial MONDO:0007788"],["Hyperlipoproteinemia, type Ib\t207750"],["Hypertriglyceridemia 2, OMIM:619324"],["Hypertriglyceridemia, transient infantile, 614480"],["Hyperlipoproteinemia, type 1D, 615947"],["Lipase deficiency, combined, 246650"],["Combined hyperlipidemia, familial \t144250","Lipoprotein lipase deficiency, 238600"],["hypertriglyceridaemia"]],["BOTH monoallelic and biallelic (but BIALLELIC mutations cause a more SEVERE disease form), autosomal or pseudoautosomal","BIALLELIC, autosomal or pseudoautosomal","MONOALLELIC, autosomal or pseudoautosomal, imprinted status unknown","BIALLELIC, autosomal or pseudoautosomal","BIALLELIC, autosomal or pseudoautosomal","BIALLELIC, autosomal or pseudoautosomal","BOTH monoallelic and biallelic, autosomal or pseudoautosomal","MONOALLELIC, autosomal or pseudoautosomal, imprinted status unknown"],[[],[],[],[],[],[],[],[]],[null,null,null,null,null,null,null,null],[["RAP3","APOA-V"],[],["CREB-H"],[],["LOC338328","GPI-HBP1"],["FLJ12681","JFP11","FLJ22302","TMEM112A"],[],["KIAA0279","MEGF3","Flamingo1","CDHF10","ADGRC2"]],["protein_coding","protein_coding","protein_coding","protein_coding",null,"protein_coding","protein_coding","protein_coding"],["HGNC:17288","HGNC:609","HGNC:18855","HGNC:4455","HGNC:24945","HGNC:14154","HGNC:6677","HGNC:3231"],["apolipoprotein A5","apolipoprotein C2","cAMP responsive element binding protein 3 like 3","glycerol-3-phosphate dehydrogenase 1","glycosylphosphatidylinositol anchored high density lipoprotein binding protein 1","lipase maturation factor 1","lipoprotein lipase","cadherin EGF LAG seven-pass G-type receptor 2"],[["606368"],["608083"],["611998"],["138420"],["612757"],["611761"],["609708"],["604265"]],[null,null,null,null,"endothelial cell LPL transporter",null,null,"adhesion G protein-coupled receptor C2"],["APOA5","APOC2","CREB3L3","GPD1","GPIHBP1","LMF1","LPL","CELSR2"],["APOA5","APOC2","CREB3L3","GPD1","GPIHBP1","LMF1","LPL","CELSR2"],["2017-11-03T00:00:00","2017-11-03T00:00:00","2017-11-03","2017-11-03T00:00:00","2017-11-03T00:00:00","2017-11-03T00:00:00","2017-11-03T00:00:00","2017-11-03T00:00:00"],["2001-12-11","2001-06-22","2003-12-09","2001-06-22","2008-02-07","2007-11-29","2001-06-22","1998-03-25"],["11:116660083-116663136","19:45449243-45452822","19:4153598-4173051","12:50497602-50505102","8:144295068-144299044","16:903634-1031318","8:19759228-19824769","1:109792641-109818372"],["ENSG00000110243","ENSG00000234906","ENSG00000060566","ENSG00000167588","ENSG00000182851","ENSG00000103227","ENSG00000175445","ENSG00000143126"],["11:116789367-116792420","19:44945982-44949565","19:4153601-4173054","12:50103819-50111319","8:143213193-143217170","16:853634-981318","8:19901717-19967258","1:109250019-109275750"],["ENSG00000110243","ENSG00000234906","ENSG00000060566","ENSG00000167588","ENSG00000277494","ENSG00000103227","ENSG00000175445","ENSG00000143126"],[228,228,228,228,228,228,228,228],["57eab3658f620356ef5e4010","57eab3658f620356ef5e4010","57eab3658f620356ef5e4010","57eab3658f620356ef5e4010","57eab3658f620356ef5e4010","57eab3658f620356ef5e4010","57eab3658f620356ef5e4010","57eab3658f620356ef5e4010"],["Cardiovascular disorders","Cardiovascular disorders","Cardiovascular disorders","Cardiovascular disorders","Cardiovascular disorders","Cardiovascular disorders","Cardiovascular disorders","Cardiovascular disorders"],["Arteriopathies","Arteriopathies","Arteriopathies","Arteriopathies","Arteriopathies","Arteriopathies","Arteriopathies","Arteriopathies"],["public","public","public","public","public","public","public","public"],["1.17","1.17","1.17","1.17","1.17","1.17","1.17","1.17"],["2022-08-03T09:45:17.972684Z","2022-08-03T09:45:17.972684Z","2022-08-03T09:45:17.972684Z","2022-08-03T09:45:17.972684Z","2022-08-03T09:45:17.972684Z","2022-08-03T09:45:17.972684Z","2022-08-03T09:45:17.972684Z","2022-08-03T09:45:17.972684Z"],[null,null,null,null,null,null,null,null],[null,null,null,null,null,null,null,null],[null,null,null,null,null,null,null,null],[null,null,null,null,null,null,null,null],[null,null,null,null,null,null,null,null],[null,null,null,null,null,null,null,null],[null,null,null,null,null,null,null,null],[null,null,null,null,null,null,null,null],[null,null,null,null,null,null,null,null],[null,null,null,null,null,null,null,null],[null,null,null,null,null,null,null,null],[null,null,null,null,null,null,null,null],[null,null,null,null,null,null,null,null],[null,null,null,null,null,null,null,null],[null,null,null,null,null,null,null,null],[null,null,null,null,null,null,null,null],[null,null,null,null,null,null,null,null],[null,null,null,null,null,null,null,null]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>panel_id<\/th>\n      <th>entity_name<\/th>\n      <th>name<\/th>\n      <th>entity_type<\/th>\n      <th>confidence_level<\/th>\n      <th>penetrance<\/th>\n      <th>mode_of_pathogenicity<\/th>\n      <th>publications<\/th>\n      <th>evidence<\/th>\n      <th>phenotypes<\/th>\n      <th>mode_of_inheritance<\/th>\n      <th>tags<\/th>\n      <th>transcript<\/th>\n      <th>gene_data.alias<\/th>\n      <th>gene_data.biotype<\/th>\n      <th>gene_data.hgnc_id<\/th>\n      <th>gene_data.gene_name<\/th>\n      <th>gene_data.omim_gene<\/th>\n      <th>gene_data.alias_name<\/th>\n      <th>gene_data.gene_symbol<\/th>\n      <th>gene_data.hgnc_symbol<\/th>\n      <th>gene_data.hgnc_release<\/th>\n      <th>gene_data.hgnc_date_symbol_changed<\/th>\n      <th>gene_data.ensembl_genes.GRch37.82.location<\/th>\n      <th>gene_data.ensembl_genes.GRch37.82.ensembl_id<\/th>\n      <th>gene_data.ensembl_genes.GRch38.90.location<\/th>\n      <th>gene_data.ensembl_genes.GRch38.90.ensembl_id<\/th>\n      <th>id<\/th>\n      <th>hash_id<\/th>\n      <th>disease_group<\/th>\n      <th>disease_sub_group<\/th>\n      <th>status<\/th>\n      <th>version<\/th>\n      <th>version_created<\/th>\n      <th>gene_data.ensembl_genes.GRch38.107.location<\/th>\n      <th>gene_data.ensembl_genes.GRch38.107.ensembl_id<\/th>\n      <th>gene_data.ensembl_genes<\/th>\n      <th>gene_data.ensembl_genes.GRch38.102.location<\/th>\n      <th>gene_data.ensembl_genes.GRch38.102.ensembl_id<\/th>\n      <th>panel.id<\/th>\n      <th>panel.hash_id<\/th>\n      <th>panel.name<\/th>\n      <th>panel.disease_group<\/th>\n      <th>panel.disease_sub_group<\/th>\n      <th>panel.status<\/th>\n      <th>panel.version<\/th>\n      <th>panel.version_created<\/th>\n      <th>panel.relevant_disorders<\/th>\n      <th>panel.types<\/th>\n      <th>panel.stats.number_of_genes<\/th>\n      <th>panel.stats.number_of_strs<\/th>\n      <th>panel.stats.number_of_regions<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Bfrtip","buttons":["copy","csv","excel","pdf","print"],"pageLength":20,"language":{"search":"","searchPlaceholder":"Enter natural language query..."},"initComplete":"\n        function(settings, json) {\n          var filter = $('div.dataTables_filter');\n          filter.css({'width': '100%'});\n          filter.find('input').css({\n            'width': '100%',\n            'border': '1px solid #ccc',\n            'padding': '8px',\n            'border-radius': '4px',\n            'box-shadow': '0 1px 3px rgba(0,0,0,0.2)',\n            'font-size': '14px',\n            'font-family': 'system-ui, -apple-system, BlinkMacSystemFont, \"Segoe UI\", Roboto, Oxygen, Ubuntu, Cantarell, \"Open Sans\", \"Helvetica Neue\", sans-serif'\n          });\n          $('table.dataTable').css({\n            'font-family': 'system-ui, -apple-system, BlinkMacSystemFont, \"Segoe UI\", Roboto, Oxygen, Ubuntu, Cantarell, \"Open Sans\", \"Helvetica Neue\", sans-serif'\n          });\n        }\n      ","columnDefs":[{"className":"dt-right","targets":[1,28,40,50,51,52]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"panel_id","targets":1},{"name":"entity_name","targets":2},{"name":"name","targets":3},{"name":"entity_type","targets":4},{"name":"confidence_level","targets":5},{"name":"penetrance","targets":6},{"name":"mode_of_pathogenicity","targets":7},{"name":"publications","targets":8},{"name":"evidence","targets":9},{"name":"phenotypes","targets":10},{"name":"mode_of_inheritance","targets":11},{"name":"tags","targets":12},{"name":"transcript","targets":13},{"name":"gene_data.alias","targets":14},{"name":"gene_data.biotype","targets":15},{"name":"gene_data.hgnc_id","targets":16},{"name":"gene_data.gene_name","targets":17},{"name":"gene_data.omim_gene","targets":18},{"name":"gene_data.alias_name","targets":19},{"name":"gene_data.gene_symbol","targets":20},{"name":"gene_data.hgnc_symbol","targets":21},{"name":"gene_data.hgnc_release","targets":22},{"name":"gene_data.hgnc_date_symbol_changed","targets":23},{"name":"gene_data.ensembl_genes.GRch37.82.location","targets":24},{"name":"gene_data.ensembl_genes.GRch37.82.ensembl_id","targets":25},{"name":"gene_data.ensembl_genes.GRch38.90.location","targets":26},{"name":"gene_data.ensembl_genes.GRch38.90.ensembl_id","targets":27},{"name":"id","targets":28},{"name":"hash_id","targets":29},{"name":"disease_group","targets":30},{"name":"disease_sub_group","targets":31},{"name":"status","targets":32},{"name":"version","targets":33},{"name":"version_created","targets":34},{"name":"gene_data.ensembl_genes.GRch38.107.location","targets":35},{"name":"gene_data.ensembl_genes.GRch38.107.ensembl_id","targets":36},{"name":"gene_data.ensembl_genes","targets":37},{"name":"gene_data.ensembl_genes.GRch38.102.location","targets":38},{"name":"gene_data.ensembl_genes.GRch38.102.ensembl_id","targets":39},{"name":"panel.id","targets":40},{"name":"panel.hash_id","targets":41},{"name":"panel.name","targets":42},{"name":"panel.disease_group","targets":43},{"name":"panel.disease_sub_group","targets":44},{"name":"panel.status","targets":45},{"name":"panel.version","targets":46},{"name":"panel.version_created","targets":47},{"name":"panel.relevant_disorders","targets":48},{"name":"panel.types","targets":49},{"name":"panel.stats.number_of_genes","targets":50},{"name":"panel.stats.number_of_strs","targets":51},{"name":"panel.stats.number_of_regions","targets":52}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[10,20,25,50,100]},"selection":{"mode":"multiple","selected":null,"target":"row","selectable":null}},"evals":["options.initComplete"],"jsHooks":[]}</script>
<script type="application/htmlwidget-sizing" data-for="htmlwidget-21ad53bf65548e180d4d">{"viewer":{"width":"100%","height":"auto","padding":15,"fill":true},"browser":{"width":"100%","height":"auto","padding":40,"fill":false}}</script>
</body>
</html>
