<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8"/>
<style>body{background-color:white;}</style>
<link href="common_lib/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet" />
<script src="common_lib/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="common_lib/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="common_lib/datatables-binding-0.33/datatables.js"></script>
<script src="common_lib/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="common_lib/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="common_lib/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="common_lib/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<script src="common_lib/jszip-1.13.6/jszip.min.js"></script>
<script src="common_lib/pdfmake-1.13.6/pdfmake.js"></script>
<script src="common_lib/pdfmake-1.13.6/vfs_fonts.js"></script>
<link href="common_lib/dt-ext-buttons-1.13.6/css/buttons.dataTables.min.css" rel="stylesheet" />
<script src="common_lib/dt-ext-buttons-1.13.6/js/dataTables.buttons.min.js"></script>
<script src="common_lib/dt-ext-buttons-1.13.6/js/buttons.html5.min.js"></script>
<script src="common_lib/dt-ext-buttons-1.13.6/js/buttons.colVis.min.js"></script>
<script src="common_lib/dt-ext-buttons-1.13.6/js/buttons.print.min.js"></script>
<link href="common_lib/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet" />
<script src="common_lib/crosstalk-1.2.1/js/crosstalk.min.js"></script>
  <title>datatables</title>
</head>
<body>
<div id="htmlwidget_container">
  <div class="datatables html-widget html-fill-item" id="htmlwidget-15a3201bc4584fb78d52" style="width:100%;height:auto;"></div>
</div>
<script type="application/json" data-for="htmlwidget-15a3201bc4584fb78d52">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"data":[["1","2","3","4","5","6","7"],[930,930,930,930,930,930,930],["APC","APOB","LDLR","MEN1","PCSK9","RET","VHL"],["Additional findings health related - children","Additional findings health related - children","Additional findings health related - children","Additional findings health related - children","Additional findings health related - children","Additional findings health related - children","Additional findings health related - children"],["gene","gene","gene","gene","gene","gene","gene"],["3","3","3","3","3","3","3"],[null,null,null,null,null,null,null],["","Loss-of-function variants (as defined in pop up message) DO NOT cause this phenotype - please provide details in the comments","","","Loss-of-function variants (as defined in pop up message) DO NOT cause this phenotype - please provide details in the comments","Loss-of-function variants (as defined in pop up message) DO NOT cause this phenotype - please provide details in the comments",""],[[],[],[],[],[],[],[]],[["Expert Review Green","Expert list"],["Other","Expert Review Green"],["Expert Review Green","Expert list"],["Expert Review Green","Expert list"],["Other","Expert Review Green"],["Other","Expert Review Green"],["Other","Expert Review Green"]],[["Familial Adenomatous Polyposis","Adult and child","Bowel cancer predisposition"],["Adult and child","Familial hypercholesterolaemia"],["Adult and child","Familial hypercholesterolaemia"],["Other cancer predisposition","Adult and child","Myltiple endocrine Neoplasia Type 1"],["Adult and child","Familial hypercholesterolaemia"],["Other cancer predisposition","Myltiple endocrine Neoplasia Type 2","Adult and child"],["Other cancer predisposition","Von Hippel Lindau Syndrome","Adult and child"]],["MONOALLELIC, autosomal or pseudoautosomal, imprinted status unknown","MONOALLELIC, autosomal or pseudoautosomal, NOT imprinted","MONOALLELIC, autosomal or pseudoautosomal, NOT imprinted","MONOALLELIC, autosomal or pseudoautosomal, imprinted status unknown","MONOALLELIC, autosomal or pseudoautosomal, imprinted status unknown","MONOALLELIC, autosomal or pseudoautosomal, imprinted status unknown","MONOALLELIC, autosomal or pseudoautosomal, imprinted status unknown"],[[],[],[],[],[],[],[]],[["ENST00000257430.9","NM_000038.5"],["ENST00000233242.5","NM_000384.2"],["ENST00000558518.5","NM_000527.4"],["ENST00000312049.10","NM_130799.2"],["ENST00000302118.5","NM_174936.3"],["ENST00000355710.8","NM_020975.4"],["ENST00000256474.2","NM_000551.3"]],[["DP2","DP3","DP2.5","PPP1R46"],[],["LDLCQ2"],[],["NARC-1","FH3"],["PTC","CDHF12","RET51","CDHR16"],["VHL1"]],["protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding"],["HGNC:583","HGNC:603","HGNC:6547","HGNC:7010","HGNC:20001","HGNC:9967","HGNC:12687"],["APC, WNT signaling pathway regulator","apolipoprotein B","low density lipoprotein receptor","menin 1","proprotein convertase subtilisin/kexin type 9","ret proto-oncogene","von Hippel-Lindau tumor suppressor"],[["611731"],["107730"],["606945"],["613733"],["607786"],["164761"],["608537"]],["protein phosphatase 1, regulatory subunit 46",null,"familial hypercholesterolemia","menin",null,["cadherin-related family member 16","RET receptor tyrosine kinase","rearranged during transfection"],null],["APC","APOB","LDLR","MEN1","PCSK9","RET","VHL"],["APC","APOB","LDLR","MEN1","PCSK9","RET","VHL"],["2017-11-03","2017-11-03","2017-11-03","2017-11-03","2017-11-03","2017-11-03","2017-11-03"],["1986-01-01","2001-06-22","1986-01-01","2001-06-22","2003-05-13","1990-07-15","1986-01-01"],["5:112043195-112181936","2:21224301-21266945","19:11200038-11244492","11:64570982-64578766","1:55505221-55530525","10:43572475-43625799","3:10182692-10193904"],["ENSG00000134982","ENSG00000084674","ENSG00000130164","ENSG00000133895","ENSG00000169174","ENSG00000165731","ENSG00000134086"],["5:112707498-112846239","2:21001429-21044073","19:11089362-11133816","11:64803510-64811294","1:55039548-55064852","10:43077027-43130351","3:10141008-10152220"],["ENSG00000134982","ENSG00000084674","ENSG00000130164","ENSG00000133895","ENSG00000169174","ENSG00000165731","ENSG00000134086"],[930,930,930,930,930,930,930],[null,null,null,null,null,null,null],["","","","","","",""],["","","","","","",""],["public","public","public","public","public","public","public"],["1.0","1.0","1.0","1.0","1.0","1.0","1.0"],["2021-04-07T10:23:11.573745Z","2021-04-07T10:23:11.573745Z","2021-04-07T10:23:11.573745Z","2021-04-07T10:23:11.573745Z","2021-04-07T10:23:11.573745Z","2021-04-07T10:23:11.573745Z","2021-04-07T10:23:11.573745Z"],[null,null,null,null,null,null,null],[null,null,null,null,null,null,null],[null,null,null,null,null,null,null],[null,null,null,null,null,null,null],[null,null,null,null,null,null,null],[null,null,null,null,null,null,null],[null,null,null,null,null,null,null],[null,null,null,null,null,null,null],[null,null,null,null,null,null,null],[null,null,null,null,null,null,null],[null,null,null,null,null,null,null],[null,null,null,null,null,null,null],[null,null,null,null,null,null,null],[null,null,null,null,null,null,null],[null,null,null,null,null,null,null],[null,null,null,null,null,null,null],[null,null,null,null,null,null,null],[null,null,null,null,null,null,null]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>panel_id<\/th>\n      <th>entity_name<\/th>\n      <th>name<\/th>\n      <th>entity_type<\/th>\n      <th>confidence_level<\/th>\n      <th>penetrance<\/th>\n      <th>mode_of_pathogenicity<\/th>\n      <th>publications<\/th>\n      <th>evidence<\/th>\n      <th>phenotypes<\/th>\n      <th>mode_of_inheritance<\/th>\n      <th>tags<\/th>\n      <th>transcript<\/th>\n      <th>gene_data.alias<\/th>\n      <th>gene_data.biotype<\/th>\n      <th>gene_data.hgnc_id<\/th>\n      <th>gene_data.gene_name<\/th>\n      <th>gene_data.omim_gene<\/th>\n      <th>gene_data.alias_name<\/th>\n      <th>gene_data.gene_symbol<\/th>\n      <th>gene_data.hgnc_symbol<\/th>\n      <th>gene_data.hgnc_release<\/th>\n      <th>gene_data.hgnc_date_symbol_changed<\/th>\n      <th>gene_data.ensembl_genes.GRch37.82.location<\/th>\n      <th>gene_data.ensembl_genes.GRch37.82.ensembl_id<\/th>\n      <th>gene_data.ensembl_genes.GRch38.90.location<\/th>\n      <th>gene_data.ensembl_genes.GRch38.90.ensembl_id<\/th>\n      <th>id<\/th>\n      <th>hash_id<\/th>\n      <th>disease_group<\/th>\n      <th>disease_sub_group<\/th>\n      <th>status<\/th>\n      <th>version<\/th>\n      <th>version_created<\/th>\n      <th>gene_data.ensembl_genes.GRch38.107.location<\/th>\n      <th>gene_data.ensembl_genes.GRch38.107.ensembl_id<\/th>\n      <th>gene_data.ensembl_genes<\/th>\n      <th>gene_data.ensembl_genes.GRch38.102.location<\/th>\n      <th>gene_data.ensembl_genes.GRch38.102.ensembl_id<\/th>\n      <th>panel.id<\/th>\n      <th>panel.hash_id<\/th>\n      <th>panel.name<\/th>\n      <th>panel.disease_group<\/th>\n      <th>panel.disease_sub_group<\/th>\n      <th>panel.status<\/th>\n      <th>panel.version<\/th>\n      <th>panel.version_created<\/th>\n      <th>panel.relevant_disorders<\/th>\n      <th>panel.types<\/th>\n      <th>panel.stats.number_of_genes<\/th>\n      <th>panel.stats.number_of_strs<\/th>\n      <th>panel.stats.number_of_regions<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Bfrtip","buttons":["copy","csv","excel","pdf","print"],"pageLength":20,"language":{"search":"","searchPlaceholder":"Enter natural language query..."},"initComplete":"\n        function(settings, json) {\n          var filter = $('div.dataTables_filter');\n          filter.css({'width': '100%'});\n          filter.find('input').css({\n            'width': '100%',\n            'border': '1px solid #ccc',\n            'padding': '8px',\n            'border-radius': '4px',\n            'box-shadow': '0 1px 3px rgba(0,0,0,0.2)',\n            'font-size': '14px',\n            'font-family': 'system-ui, -apple-system, BlinkMacSystemFont, \"Segoe UI\", Roboto, Oxygen, Ubuntu, Cantarell, \"Open Sans\", \"Helvetica Neue\", sans-serif'\n          });\n          $('table.dataTable').css({\n            'font-family': 'system-ui, -apple-system, BlinkMacSystemFont, \"Segoe UI\", Roboto, Oxygen, Ubuntu, Cantarell, \"Open Sans\", \"Helvetica Neue\", sans-serif'\n          });\n        }\n      ","columnDefs":[{"className":"dt-right","targets":[1,28,40,50,51,52]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"panel_id","targets":1},{"name":"entity_name","targets":2},{"name":"name","targets":3},{"name":"entity_type","targets":4},{"name":"confidence_level","targets":5},{"name":"penetrance","targets":6},{"name":"mode_of_pathogenicity","targets":7},{"name":"publications","targets":8},{"name":"evidence","targets":9},{"name":"phenotypes","targets":10},{"name":"mode_of_inheritance","targets":11},{"name":"tags","targets":12},{"name":"transcript","targets":13},{"name":"gene_data.alias","targets":14},{"name":"gene_data.biotype","targets":15},{"name":"gene_data.hgnc_id","targets":16},{"name":"gene_data.gene_name","targets":17},{"name":"gene_data.omim_gene","targets":18},{"name":"gene_data.alias_name","targets":19},{"name":"gene_data.gene_symbol","targets":20},{"name":"gene_data.hgnc_symbol","targets":21},{"name":"gene_data.hgnc_release","targets":22},{"name":"gene_data.hgnc_date_symbol_changed","targets":23},{"name":"gene_data.ensembl_genes.GRch37.82.location","targets":24},{"name":"gene_data.ensembl_genes.GRch37.82.ensembl_id","targets":25},{"name":"gene_data.ensembl_genes.GRch38.90.location","targets":26},{"name":"gene_data.ensembl_genes.GRch38.90.ensembl_id","targets":27},{"name":"id","targets":28},{"name":"hash_id","targets":29},{"name":"disease_group","targets":30},{"name":"disease_sub_group","targets":31},{"name":"status","targets":32},{"name":"version","targets":33},{"name":"version_created","targets":34},{"name":"gene_data.ensembl_genes.GRch38.107.location","targets":35},{"name":"gene_data.ensembl_genes.GRch38.107.ensembl_id","targets":36},{"name":"gene_data.ensembl_genes","targets":37},{"name":"gene_data.ensembl_genes.GRch38.102.location","targets":38},{"name":"gene_data.ensembl_genes.GRch38.102.ensembl_id","targets":39},{"name":"panel.id","targets":40},{"name":"panel.hash_id","targets":41},{"name":"panel.name","targets":42},{"name":"panel.disease_group","targets":43},{"name":"panel.disease_sub_group","targets":44},{"name":"panel.status","targets":45},{"name":"panel.version","targets":46},{"name":"panel.version_created","targets":47},{"name":"panel.relevant_disorders","targets":48},{"name":"panel.types","targets":49},{"name":"panel.stats.number_of_genes","targets":50},{"name":"panel.stats.number_of_strs","targets":51},{"name":"panel.stats.number_of_regions","targets":52}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[10,20,25,50,100]},"selection":{"mode":"multiple","selected":null,"target":"row","selectable":null}},"evals":["options.initComplete"],"jsHooks":[]}</script>
<script type="application/htmlwidget-sizing" data-for="htmlwidget-15a3201bc4584fb78d52">{"viewer":{"width":"100%","height":"auto","padding":15,"fill":true},"browser":{"width":"100%","height":"auto","padding":40,"fill":false}}</script>
</body>
</html>
